spring-integration-http/src/main/java/org/springframework/integration/http/outbound/HttpRequestExecutingMessageHandler.java

```diff
@@ -79,40 +52,12 @@
  * @author Gunnar Hillert
  * @author Artem Bilan
  * @author Wallace Wadge
+ * @author Shiliang Li
  * @since 2.0
  */
-public class HttpRequestExecutingMessageHandler extends AbstractReplyProducingMessageHandler {
-
-	private final Map<String, Expression> uriVariableExpressions = new HashMap<>();
-
+public class HttpRequestExecutingMessageHandler extends AbstractHttpRequestExecutingMessageHandler {
```

Comment:
I used `git merge` instead of `git rebase` based on the instruction on https://github.com/spring-projects/spring-integration/blob/master/CONTRIBUTING.adoc which mentioned:

> Note You cannot rebase if you have already pushed your branch to your remote because you’d be rewriting history (see 'The Perils of Rebasing' in the article). If you rebase by mistake, you can undo it as discussed in this stackoverflow discussion. Once you have published your branch, you need to merge in the master rather than rebasing.

And when I do `git rebase`, I see old implementation instead of the latest change, many files are overwritten, like the http.adoc and org.springframework.integration.http.dsl.http class, etc. Do you have any suggestion about how to deal with this situation?